src/pilot/ovs_dpdk_setup.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python

```

Comment:
I really feel that this piece of code should be eliminated.  This is because this code makes it so that DPDK can only be used with R630 and R730 servers, and it also hard codes the NICs for these servers into one configuration.  We have had requests to make the solution more flexible, and this piece of code moves us in the opposite direction.

Reply:
Removing this hardcoding is part of the planned activities. It's currently in-progress.

Reply:
Would it be better to check if the server that we have chosen to work with is supported by a kind of support matrix? Maybe we can enforce this check by use of our profiles.

Reply:
Well, I think we want users to be able to use this with any hardware.  It may not work if it's hardware we haven't tested with, but chances are good that it will work.